src/OrbitGl/TimelineUi.cpp

```diff
@@ -76,10 +76,12 @@ void TimelineUi::RenderLabel(Batcher& batcher, TextRenderer& text_renderer, uint
                          TextRenderer::HAlign::Left, TextRenderer::VAlign::Middle},
                         /*out_text_pos=*/&pos, /*out_text_size=*/&size);
 
-  // Box behind the label to hide the ticks behind it.
-  size[0] += 2.f * kLabelsPadding;
+  // Box behind the label to hide the ticks behind it. We make the left padding a pixel smaller, so
+  // they don't incercept with the proper tick.
+  constexpr float kLeftPadding = kLabelsPadding - 1.f;
```

Comment:
This should be clamped so that kLeftPadding is never negative, right?

Reply:
Actually, wouldn't simply drawing the label one pixel to the right achieve a similar result?

Reply:
Yes, I thought about that. I just didn't want to move it to the right because the horizontal space is gold for us (labels intersection), but now that you mention I think you might be right. I will ask Carsten since he is in Munich today and keep you posted :)

Reply:
Carsten agrees, so I push your solution.